src/ui/RecipeTitleCard.tsx

```diff
@@ -0,0 +1,103 @@
+import styled from '@emotion/styled';
+import Flex from './Flex';
+
+const StyledFooter = styled.footer`
+  display: flex;
+  gap: 8px;
+`;
+
+const BydelName = styled.footer`
+  font-size: 17px;
+  line-height: 20.74px;
+  font-weight: 600;
+  // font-family-sneak;
+`;
+
+const RecipeName = styled.label`
+  font-weight: 600;
+  text-transform: uppercase;
+  font-size: 30px;
+  line-height: 34.2px;
+`;
+const AuthorName = styled.label`
+  width: 277px;
+  font-weight: 400;
+  text-transform: uppercase;
```

Comment:
i prefer to use javascript for this (toUpperCase) but just because i'm much more comfortable in JS than CSS. feel free to leave this as is!